advocacy_docs/partner_docs/DBeaverGuide/04-ConfiguringTheIntegration.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: 'Configuring the Integration'
+description: 'Walkthrough on configuring DBeaver'
+---
+
+Implementing DBeaver with EDB Postgres Advanced Server and EDB Postgres Extended requires the following components: 
+
+- A running EDB Postgres Advanced Server or EDB Postgres Extended instance.
+- DBeaver application installed on a system where you will manage the running database.
+
+## EDB Postgres Advanced Server Configuration
+
+1. Launch DBeaver on your system. 
+2. Go to the `Database` drop down menu in the top left, and select `New Database Connection`. In the `All` tab, select `EnterpriseDB` and click `Next`.    
+
+<p align="center">
+<img src="Images/NewDatabaseConnection.png">
+</p>
+
```

Comment:
Spacing between the Image - note - image  should be even.  The first image to note is much larger than the space from the note to the following image.

Reply:
To fix the spacing, I updated the image to use in
https://docs.google.com/document/d/17ijHoiaruCPFxOsi6ZYpWM839G9Dr6ZcDsTs-jcJDPs/edit#
It should remove some of the white space.